src/browser/services/fetch-client.js

```diff
@@ -2,8 +2,10 @@ import { Client, createError } from '../../core';
 import isHtml from 'is-html';
 
 export default class FetchClient extends Client {
-  request(url, method = 'get', body, headers = {}) {
-    const options = {
+  request(url, body, headers = {}, options = {}) {
```

Comment:
The method is passed to sdk as an option and then we chose to separate it into a different variable. This way and to avoid increase the argument size on this method, I opt to remove it and start using the one on the `options` object